tests/unit/parser.js

```diff
@@ -6935,6 +6935,7 @@ exports.testStrictDirectiveASI = function (test) {
     .test("'use strict'\n(function fn() {})();", options);
 
   TestRun(test, 5)
+    .addError(2, "Missing \"use strict\" statement.")
```

Comment:
no, I still don't understand the warning that is being emit here. Why is this warning being expected?